### PR TITLE
Add automation to the Orc Warmask feat

### PR DESCRIPTION
**Pull request by TikaelSol**

### CODE REVIEW

File: packs/data/equipment.db/orc-warmask.json [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        },
+        "price": {
+            "value": {}
+        },
+        "quantity": 1,
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.SpecificRule.Orc.OrceWarmask.Gods",
```
Comment (simonward) [inline packs/data/equipment.db/orc-warmask.json:40]: Should be "PF2E.SpecificRule.Orc.Warmask.Gods"

File: packs/data/equipment.db/orc-warmask.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                        "value": "religion"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.Orc.OrceWarmask.Land",
```
Comment (simonward) [inline packs/data/equipment.db/orc-warmask.json:44]: "PF2E.SpecificRule.Orc.Warmask.Land"

File: packs/data/equipment.db/orc-warmask.json [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+                        "value": "nature"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.Orc.OrceWarmask.Magic",
```
Comment (simonward) [inline packs/data/equipment.db/orc-warmask.json:48]: "PF2E.SpecificRule.Orc.Warmask.Magic"

File: packs/data/equipment.db/orc-warmask.json [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+                        "value": "arcana"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.Orc.OrceWarmask.Unknown",
```
Comment (simonward) [inline packs/data/equipment.db/orc-warmask.json:52]: "PF2E.SpecificRule.Orc.Warmask.Unknown"

File: packs/data/equipment.db/orc-warmask.json [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+                        "value": "occultism"
+                    }
+                ],
+                "flag": "tradition",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Orc.OrceWarmask.Prompt"
```
Comment (simonward) [inline packs/data/equipment.db/orc-warmask.json:58]: "PF2E.SpecificRule.Orc.Warmask.Prompt"

**Comment by simonward** [comment]
Missing changes to re-en.json

File: packs/data/equipment.db/orc-warmask.json (hunk 1 above)
Comment (TikaelSol) [reply packs/data/equipment.db/orc-warmask.json:40]: darn copy/paste

File: packs/data/equipment.db/orc-warmask.json [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+                        "value": "arcana"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.Orc.OrcWarmask.Unknown",
+                        "value": "occultism"
+                    }
+                ],
+                "flag": "tradition",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Orc.OrceWarmask.Prompt"
```
Comment (TikaelSol) [inline packs/data/equipment.db/orc-warmask.json:58]:
```suggestion
                "prompt": "PF2E.SpecificRule.Orc.OrcWarmask.Prompt"
```

**Review by simonward — DISMISSED** [review]

File: packs/data/equipment.db/orc-warmask.json [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+            "value": "Pathfinder Lost Omens: Ancestry Guide"
+        },
+        "stackGroup": null,
+        "traits": {
+            "rarity": "common",
+            "value": [
+                "invested",
+                "magical"
+            ]
+        },
```
Comment (SpartanCPA) [inline packs/data/equipment.db/orc-warmask.json:78]:
```suggestion
        "traits": {
            "rarity": "common",
            "value": [
                "divination",
                "invested",
                "magical"
            ]
        },
```

File: packs/data/equipment.db/orc-warmask.json [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+                "flag": "tradition",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Orc.OrcWarmask.Prompt"
+            },
+            {
+                "key": "FlatModifier",
+                "selector": "{item|flags.pf2e.rulesSelections.tradition}",
+                "type": "item",
+                "value": 1
+            }
```
Comment (SpartanCPA) [inline packs/data/equipment.db/orc-warmask.json:65]: This is not a default feature, you need to do extra things to activate this benefit.

Comment (TikaelSol) [reply packs/data/equipment.db/orc-warmask.json:65]: It seemed like a very specific thing to add a toggle for. It's an invested item with no benefit until you spend the money.

**Pull request merged**